### PR TITLE
Upload scraper dantri

### PR DESCRIPTION
**Pull request by quangLH195**
I have done it in last night and this time i'm pretty sure that work well bc i checked that many time, thank you guys

### CODE REVIEW

File: spider/__main__.py [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
+async def main(dest):
+    """Download vaccine images."""
+    async with AsyncClient() as client, open_nursery() as nursery:
+        nursery.start_soon(tuoitre, Path(dest), client, nursery)
+        nursery.start_soon(vnexpress, Path(dest), client, nursery)
+        nursery.start_soon(dantri, Path(dest), client, nursery)
+
+
+if __name__ == '__main__':
+    run(main, argv[1] if len(argv) > 1 else '.')
```
Comment (McSinyx) [inline spider/__main__.py]: Ehm, what's up with such a hugh diff?

**Review by McSinyx — CHANGES_REQUESTED** [review]
FYI, `python -m spider` doesn't run because you messed up the imports in `__main__.py`.  Also please check you source, the only scraped image is the portrait of the antitrust offender.

File: spider/__main__.py (hunk 1 above)
Comment (quangLH195) [reply spider/__main__.py]: oh ok, i got that, when i checked that in my computer, i should remove the dot in "from .dantri import dantri" and change by "from dantri import dantri" and the same with vnexpress and tuoitre then it worked for our code

Comment (McSinyx) [reply spider/__main__.py]: cd into the project's and `python -m spider`, what's so difficult with that

**Comment by McSinyx** [comment]
The diff is still huge and confusing :facepalm:, please

    git fetch origin main
    git restore -s main spider/__main__.py

And use an editor *that doesn't override EoLs*.

File: spider/__main__.py (hunk 1 above)
Comment (quangLH195) [reply spider/__main__.py]: in window 10, I can't use `python -m spider`, as you saw yesterday, i should have move the file into the cmd then i can run the code

Comment (McSinyx) [reply spider/__main__.py]: You can, @pdhlong is using Windows too.  Maybe you have to replace python with py but you have to figure it out.

Comment (quangLH195) [reply spider/__main__.py]: sorry, i can work with that :))  and everything work fine 
abc

Comment (McSinyx) [reply spider/__main__.py]: > Also please check you source, the only scraped image is the portrait of the antitrust offender.

**Pull request closed without merging**